bindings/dart/example/lib/data_provider/web.dart

```diff
@@ -43,3 +47,24 @@ Future<Uint8List> _fetchAsset(String url, String checksum) async {
     return Future.error('error loading asset: $url, error: $e');
   }
 }
+
+/// Injects the WASM script into the HTML and exports its methods
+/// to the `window` object of the browser.
+Future<void> injectWasmScript(Uint8List scriptBytes) {
+  final completer = Completer<void>();
+  final src =
+      Url.createObjectUrl(Blob(<Uint8List>[scriptBytes], 'text/javascript'));
+
+  context['_notifyWasmBindingsLoaded'] = () => completer.complete();
+
+  document.head!.append(document.createElement('script')
```

Comment:
not sure if i understand this correctly, the script used to be in the html body before, does injecting it into the head has the same effect?
```suggestion
  document.body!.append(document.createElement('script')
```

Reply:
In our case it should have the same effect. It would matter [if the script execution takes awhile to run or if the script depends on the DOM](https://medium.com/geekculture/where-to-put-a-script-tag-into-head-or-body-end-b5b063058e0b).